src/common/license/src/license.rs

```diff
@@ -0,0 +1,15 @@
+use serde::Deserialize;
+use serde::Serialize;
+
+pub const LICENSE_PUBLIC_KEY: &str = r#"-----BEGIN PUBLIC KEY-----
+MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEGsKCbhXU7j56VKZ7piDlLXGhud0a
+pWjW3wxSdeARerxs/BeoWK7FspDtfLaAT8iJe4YEmR0JpkRQ8foWs0ve3w==
+-----END PUBLIC KEY-----"#;
+
+#[derive(Serialize, Deserialize)]
+pub struct LicenseInfo {
+    #[serde(rename = "type")]
+    pub type_: Option<String>,
+    pub org: Option<String>,
+    pub tenants: Option<Vec<String>>,
```

Comment:
I guess just `Vec<String>` is fine in this case, as an empty vector can already represents the Empty case.

Reply:
option is needed. otherwise the input must similar to 
```json
{
  tenants: []
}
```
since the jwt-simple would not parse nil to Vec